topgg-page.htm

```diff
@@ -4,75 +4,174 @@
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <head>
         <style>
+            @import url('https://fonts.googleapis.com/css2?family=Kalam&display=swap');
+            @import url('https://fonts.googleapis.com/css2?family=Nova+Mono&display=swap');
```

Comment:
loading in external fonts tends to make things slow, even on some blazingly fast connections.
Plus, these don't seem websafe to me, unless I am blind...

----
More on that here: https://www.w3schools.com/cssref/css_websafe_fonts.asp
w3 are those guys who write the web-specs, so they know their stuff.